src/main/java/chatbot/Chatbot.java

```diff
@@ -0,0 +1,188 @@
+package chatbot;
+
+import chatbot.task.Deadline;
+import chatbot.task.Event;
+import chatbot.task.Task;
+import chatbot.task.Todo;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Chatbot {
+    private final String CHATBOT_NAME;
+    private final ArrayList<Task> taskList = new ArrayList<>();
+    private int listLength = 0;
+    private boolean isExit = false;
+    String input;
+    Task selectedItem;
+    Scanner in = new Scanner(System.in);
+
+    public Chatbot(String name) {
+        this.CHATBOT_NAME = name;
+    }
+    public void writeToFile() throws IOException {
+        FileWriter fileWriter = new FileWriter("./data/chatbot.txt");
+        for (int i = 0; i < listLength; i += 1) {
+            fileWriter.write(taskList.get(i).getData() + "\n");
+        }
+        fileWriter.close();
+    }
+    public void save() {
+        try {
+            writeToFile();
+        } catch (IOException e) {
+            System.out.println("Something went wrong: " + e.getMessage());
+        }
+    }
+    public static void printTask(Task task) {
+        System.out.println(task.getData());
+    }
+
+    public void printLength() {
+        System.out.println("You have " + this.listLength + " tasks in the list. ");
+    }
+
+    public void printResponse(String command) {
+        switch (command) {
+        case "bye":
+            System.out.println("Bye. Hope not to see you again! ");
+            return;
+        case "list":
+            if (this.listLength > 0) {
+                System.out.println("Here are the tasks in your list: ");
+            } else {
+                System.out.println("There's nothing here. ");
+            }
+            return;
+        case "unmark":
+            System.out.println("Failing to meet expectations, are we? Unmarked. ");
+            break;
+        case "mark":
+            System.out.println("Not as incompetent as I thought. Marked. ");
+            break;
+        case "todo":
```

Comment:
will be good to have // Fallthrough to indicate clear intention of not breaking out